README.md

```diff
@@ -2,6 +2,13 @@
   <img src="https://github.com/box/sdks/blob/master/images/box-dev-logo.png" alt= “box-dev-logo” width="30%" height="50%">
 </p>
 
+# Migration to v10
+
+The `sdk-gen` branch contains the generated code for the `v10` version of the Box Java SDK. This is intended primarily for developers migrating from [Box Java SDK Gen](https://github.com/box/box-java-sdk-gen) to the v10 version.
```

Comment:
How about putting here a link for sdk-gen branch instead?
`[sdk-gen](https://github.com/box/box-java-sdk/tree/sdk-gen)`